### ISSUE TITLE
Obsolete

### ISSUE THREAD

**Issue by Cinnamoonroll** [mitchellcairns/robochefs#6]
Hey there!
I'm pretty new at this and found this project to start exploring into gaming development; however, when I try to open it in Unity it says it is obsolete, even when I downloaded older versions of it... I got to open it but never make it work...

Thank you for sharing this by the way!

**Comment by fmdelvalle** [comment]
Hi, i've just downloaded it, it indeed is obsolete, but you should replace GuiText with UI.Text (and the require to a fully qualified path) and GuiImage in two files and then you're left with asset issues. All assets are accesible through Git LFS, which I found it was a prerequirement (in needs to be installed and activated BEFORE git clone). But then I find myself with an issue I cannot overcome: Github refuses to serve these files upfront:

Error downloading object: Assets/Models/DevArt/Cheese.obj (20b47fa): Smudge error: Error downloading Assets/Models/DevArt/Cheese.obj (20b47fa167894ef0c35c1e916ce922fdcf3a70734f75c5a09ef863320a6362a6): batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.

This makes it impossible to download and contribute to this project.

**Comment by MestreSales** [comment]
Hello!

I have downloaded the project and fixed those issues along with a small cast problem that originated afterwards. I am using version 2019.4.15f1; Unity handled a lot of version control and other stuff I don't really unnderstand. But it seems to work on my computer. As fmdelvalle said, now it's a problem of the OP (mitchellcairns). We can't do anything if the LFS bandwith thing isn't fixed. And also, fmdelvalle the last problem you mentioned, I downloaded the project from a link provided in another issue: https://github.com/mitchellcairns/robochefs/issues/4#issuecomment-586494416 ; and the project just had those few small problems. This link helped me fix them: https://answers.unity.com/questions/1638555/guitexture-adn-guitext-are-obsolete-standard-asset.html
Anyway, here's the working version of the project were I fixed all of the small problems mentioned: https://drive.google.com/file/d/1Dd2-aoXIHaGKS05G-smtVnqVwmFYiCVJ/view?usp=sharing

Have a great 2021!
